src/machine/machine_rp2_pins.go

```diff
@@ -1,4 +1,4 @@
-//go:build rp2040 || ae_rp2040 || badger2040 || challenger_rp2040 || feather_rp2040 || gopher_badge || kb2040 || macropad_rp2040 || nano_rp2040 || pico || qtpy_rp2040 || thingplus_rp2040 || thumby || tufty2040 || waveshare_rp2040_zero || xiao_rp2040
+//go:build rp2040 || rp2350 || ae_rp2040 || badger2040 || challenger_rp2040 || feather_rp2040 || gopher_badge || kb2040 || macropad_rp2040 || nano_rp2040 || pico || qtpy_rp2040 || thingplus_rp2040 || thumby || tufty2040 || waveshare_rp2040_zero || xiao_rp2040
```

Comment:
Seems to me that one of two things must be true about this line:

- either we need to add `pico2`
- OR we should replace the entire line with this because it should already be inherited via the json:
- 
```suggestion
//go:build rp2040 || rp2350
```

Reply:
There's a wrinkle here that will have to be solved eventually- the RP2350A chip has the same 30 GPIO+ADC channels as the RP2040, however the larger RP2350B package has 48 GPIO pins, with a different ADC mapping. Since we don't have a way to specify the RP2350 variant yet, I think your suggestion is reasonable.

If a board developer wanted to provide alternate names for them, it looks like the convention is to alias the machine definitions:
https://github.com/tinygo-org/tinygo/blob/ba1919f252a2e027581d4f061ef5b7014a372071/src/machine/board_itsybitsy-m4.go#L9
https://github.com/tinygo-org/tinygo/blob/ba1919f252a2e027581d4f061ef5b7014a372071/src/machine/board_arduino.go#L7

Reply:
It would be best if we address any superfluous tags in a more focused PR later.